bundles/framework/myplaces3/view/PlaceForm.js

```diff
@@ -390,6 +392,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.view.PlaceForm',
                             buttonType: 'button',
                             onClick: (event) => {
                                 this.dialog.close();
+                                this.cancelCallback(true, true);
```

Comment:
Refactor so cancelCallback can be called without parameters. This component shouldn't know or care what happens outside of it once the user cancels. Sending random parameters to a function that was provided as a callback makes this hard to read.